components/function-controller/README.md

```diff
@@ -66,3 +66,31 @@ To develop the component, use the formulae declared in the [generic](/common/mak
 | **WEBHOOK_DEFAULTING_MINREPLICAS**        | Value of the minReplicas which webhook should set if origin equals null                   | `1`                  |
 | **WEBHOOK_DEFAULTING_MAXREPLICAS**        | Value of the maxReplicas which webhook should set if origin equals null                   | `1`                  |
 | **WEBHOOK_DEFAULTING_RUNTIME**            | Value of the runtime which webhook should set if origin equals null                       | `nodejs14`           |
+
+## Troubleshooting
+
+### How to build and install `libgit2` binary, required by the Function Controller, on macOS
```

Comment:
Please follow the [troubleshooting template](https://github.com/kyma-project/community/blob/main/templates/resources/troubleshooting.md).